dashboard/modules/snapshot/tests/test_job_submission.py

```diff
@@ -92,5 +93,23 @@ def wait_for_job_to_fail():
     wait_for_condition(wait_for_job_to_fail, timeout=30)
 
 
+@pytest.mark.parametrize("address", [
+    "http://127.0.0.1", "https://127.0.0.1", "ray://127.0.0.1",
+    "fake_module://127.0.0.1"
+])
```

Comment:
oh, oops, can you put this in `dashboard/modules/job/tests/` instead? this is specifically for the API snapshot.